schema.graphql

```diff
@@ -260,6 +260,16 @@ type AnalysisFrameworkVisibleProjectType {
   isPrivate: Boolean!
 }
 
+input AnalysisInputType {
+  id: ID
+  title: String!
+  teamLead: ID!
+  startDate: Date
+  endDate: Date!
+  clonedFrom: ID
+  analysisPillar: [AnalysisPillarGqlInputType!]
```

Comment:
@subinasr @AdityaKhatri @sudan45 @sauravsapkota 

Looking at this behavior, should we continue with this or modify the logic?

Basically, right now, while editing Analysis, one can remove one or multiple Analysis Pillar with one click.
Let discuss more on this.

Reply:
@thenav56, I think this is fine for now. I'll create an issue to handle this behavior in the frontend.
https://github.com/the-deep/client/issues/2981

Let me know what you think.

Reply:
https://notes.tools.togglecorp.com/vvLpUkNSRa-8pJUjBbpMxg?both
@thenav56  @AdityaKhatri @subinasr  @susilnem  please check once